Formula/lua.rb

```diff
@@ -45,7 +45,8 @@ class Lua < Formula
   # sigaction provided by posix signalling power patch
   if build.with? "sigaction"
     patch do
-      url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      # url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      url "https://gist.githubusercontent.com/anonymous/99614c27bcac2aa86181fabe78c85915/raw/7abdf0234ffd38fc9094078d8128938f5e2accd4/lua-5.2.3-sig_catch.patch"
```

Comment:
Not that I need to be convinced, but this didn't really convince me. I'm aware that we have a certain ambiguity thanks to our tap-to-repository mapping, but I don't feel this rename improves on this. Mostly, I'm a bit meh on renaming repositories on a whim, as I think they should be pretty stable identifiers and the rename has the potential to cause its own share of confusion. (Thankfully redirects work very well across GitHub.)